docs/report/main.tex

```diff
@@ -127,31 +140,30 @@ \subsection{Qubits}
 \vspace{5mm}
 
 \noindent
-
-These two qubit states can be expressed in column matrices as:
+These qubit states can be expressed in column matrices as:
 \vspace{5mm}
 
 \qquad $\ket{0} = \begin{pmatrix}1 \\ 0 \end{pmatrix}$ \ and \ $\ket{1} = \begin{pmatrix} 0 \\ 1 \end{pmatrix}$
 
 \vspace{5mm}
 
 \noindent
-The state of a quantum system can be represented as the spin, energy, angular momentum and magnetic moment of an elementary particle. However for the sake simplicity and brevity the state $\ket{0}$ and $\ket{1}$ will denote spin up and spin down of an electron. 
+The state of a quantum system can be represented as the spin, energy, angular momentum and magnetic moment of an elementary particle. However for the sake of brevity the state $\ket{0}$ and $\ket{1}$ will denote spin up and spin down of the electron in this report.
 \pagebreak
 
 \subsection{Quantum Register}
 \vspace{5mm}
 
 \noindent
-The qubits in a quantum computer is stored in a quantum register. The size of the quantum register is dictated by the number of qubits. 
+The qubits in a quantum computer are stored in a quantum register. The size of the quantum register is dictated by the number of qubits. 
 \vspace{5mm}
 
 \noindent
-The vector state of the quantum register $\mathcal{H}$ with $n$ qubits can be expressed in tensor product:
+The vector state of the quantum register $\mathcal{H}$ with $n$ qubits can be expressed in tensor product\hyperlink{1}{$^1$}:
```

Comment:
for citations, you should be able to use the `\cite{}` command, with a reference to the label for that citation, and latex will figure out the citation numbering when rendering the PDF

Reply:
Yes, but ran into trouble with \cite due to conflict with the table of contents. Using \cite will create its own Reference header which is independent from the table of content numbering system. So you effective get two "References " one numbered and one not(due to \cite). So I came out with this slightly long winded method. Let me know if you can find a fix ☺️

Reply:
https://github.com/Tiernan8r/quantum_computing_project/pull/78 changes to use cite. Have to make a bit of a workaround to remove the double section headering as you said.